agent/config/config.go

```diff
@@ -355,6 +363,10 @@ const (
 	// defaultPolicyWorkerAckTimeout is the default time limit that a policy
 	// eval must be ACK'd.
 	defaultPolicyEvalAckTimeout = 5 * time.Minute
+
+	// defaultLockPath is the default path used for the lock that syncs the leader
+	// election.
+	defaultLockPath = "autoscaler/lock"
```

Comment:
nit: use the full name of the project:
```suggestion
	defaultLockPath = "nomad-autoscaler/lock"
```